src/pygambit/tests/test_behav.py

```diff
@@ -7,7 +7,7 @@ def setUp(self):
             "test_games/mixed_behavior_game.efg"
         )
         self.profile_double = self.game.mixed_behavior_profile()
-        self.profile_rational = self.game.mixed_behavior_profile(True)
+        self.profile_rational = self.game.mixed_behavior_profile(None, True)
```

Comment:
It would be useful to add a test of creating the behavior profile set to a specific distribution, exercising the new code.